### PR TITLE
Fix Japanese Good Morning(お早う -> おはよう)

### PR DESCRIPTION
**Pull request by tadashi0713**
## Hey, I just made a Pull Request!

It's more natural in Japanese 😄 
https://hinative.com/ja/questions/33582

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4594-nwrpfiweaijmcmv68msjosbyrj3pbf4y.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602f585665f817a5e9f4746b

**Comment by Fox32** [comment]
I guess we need a changeset here, @tadashi0713 can you add one?

**Comment by tadashi0713** [comment]
@Fox32 thanks, I added

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4594-nwrpfiweaijmcmv68msjosbyrj3pbf4y.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602f585665f817a5e9f4746b

File: .changeset/strong-badgers-invent.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': minor
```
Comment (OrkoHunter) [inline .changeset/strong-badgers-invent.md:2]:
```suggestion
'@backstage/plugin-catalog': patch
```

Should be patch, since there is no breaking change.

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4594-nwrpfiweaijmcmv68msjosbyrj3pbf4y.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602f585665f817a5e9f4746b

**Review by freben — APPROVED** [review]

**Pull request merged**